### ISSUE TITLE
Error with NodeJS 5: "process.argv.includes is not a function"

### ISSUE THREAD

**Issue by stephdenman** [groupe-sii/sonar-web-frontend-reporters#14]
Support for NodeJS < 6 seems to be broken in the final version 3.0.0;
```
es5 = (process.argv.includes('--es5') > -1),
                    ^
TypeError: process.argv.includes is not a function
  at Object.<anonymous> (C:\a\1\s\node_modules\sonar-web-frontend-reporters\bin\sreporter:7:23)
  at Module._compile (module.js:413:34)
  at Object.Module._extensions..js (module.js:422:10)
  at Module.load (module.js:357:32)
  at Function.Module._load (module.js:314:12)
  at Function.Module.runMain (module.js:447:10)
  at startup (node.js:148:18)
  at node.js:405:3
```

**Comment by ValentinGot** [comment]
Arf yeah, i'm on it =)

**Comment by ValentinGot** [comment]
Hi @stevedenman,

It has been fixed on 07ee4c1, we are waiting to fix some other issues before releasing it.

**Comment by stephdenman** [comment]
@ValentinGot No problem, I can carry on using beta 2. Thanks.

**Comment by ValentinGot** [comment]
Ok great :)

**Comment by ValentinGot** [comment]
Released in **3.1.0**.
